src/components/ReduxPractice/Counter.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { increment, decrement } from './actions';
+
+export default () => {
+   const counter = useSelector(state => state.counter);
+   const isLogged = useSelector(state => state.isLogged);
+   const dispatch = useDispatch();
```

Comment:
Nice we'll be using these hooks a lot because all of our components are functional